config/routes.rb

```diff
@@ -11,19 +11,19 @@
   resources :named_contacts, only: :create
 
   namespace :anonymous_feedback do
-    get 'problem_reports', to: redirect {|p, req| req.params[:path] ? "/anonymous_feedback?path=" + req.params[:path] : "/anonymous_feedback"}
-
-    namespace :problem_reports do
-      get :explore, to: redirect("/anonymous_feedback/explore"), format: false
-    end
-
     get :explore, to: "explore#new", format: false
     post :explore, to: "explore#create", format: false
 
     resources :organisations, only: :show, param: :slug, format: false
 
     resources :export_requests, only: [:create, :show], format: false
     resources :global_export_requests, only: [:create], format: false
+
+    resources :problem_reports, only: [:index] do
+      collection do
+        put :review
```

Comment:
If it's only this route, we can also use `put :review, on: :collection` to one-line it. Nit.